content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -246,7 +246,7 @@ Lars Kellogg-Stedman.
 
 ### Multus (a Multi Network plugin)
 
-[Multus](https://github.com/Intel-Corp/multus-cni) is a Multi CNI plugin to support the Multi Networking feature in Kubernetes using CRD based network objects in Kubernetes.
+Multus is a Multi CNI plugin to support the Multi Networking feature in Kubernetes using CRD based network objects in Kubernetes.
```

Comment:
BTW https://github.com/k8snetworkplumbingwg/multus-cni might be the new home for that. I haven't checked.